AirSkyBoat/AirSkyBoat
# Update flash.lua

Flash should use MND as its attribute as it is a white magic enfeebling spell.



**_I affirm:_**

## What does this pull request do?
Changed the attribute of Flash from INT to MND.


## Steps to test these changes
1. Uncomment lines 617-620 in 'function getMagicResist' in scripts/globals/magic.lua to display resist chance.
2. Use Flash on a target with high dINT and low dINT while holding dMND constant. Resistance should not change.
3. Use Flash on a target with high dMND and low dMND while holding dINT constant. Resistance should be lower with higher caster MND.
